### PR TITLE
Update README.mdkooop

### PR DESCRIPTION
**Pull request by papis801**
Koop

### CODE REVIEW

**Comment by papis801** [comment]
Jiiiop

**Comment by papis801** [comment]
Ooopiopp

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-- 👋 Hi, I’m @papis801
+Kjoop- 👋 Hi, I’m @papis801
```
Comment (Gutodotado24) [inline README.md:1]: Ci8vIENvcHlyaWdodCAyMDE1IHRoZSBWOCBwcm9qZWN0IGF1dGhvcnMuIEFsbCByaWdodHMgcmVzZXJ2ZWQuCi8vIFVzZSBvZiB0aGlzIHNvdXJjZSBjb2RlIGlzIGdvdmVybmVkIGJ5IGEgQlNELXN0eWxlIGxpY2Vuc2UgdGhhdCBjYW4gYmUKLy8gZm91bmQgaW4gdGhlIExJQ0VOU0UgZmlsZS4KCiNpbmNsdWRlICJzcmMvY29tcGlsZXIvYWNjZXNzLWluZm8uaCIKCiNpbmNsdWRlIDxvc3RyZWFtPgoKI2luY2x1ZGUgInNyYy9idWlsdGlucy9hY2Nlc3NvcnMuaCIKI2luY2x1ZGUgInNyYy9jb21waWxlci9jb21waWxhdGlvbi1kZXBlbmRlbmNpZXMuaCIKI2luY2x1ZGUgInNyYy9jb21waWxlci9zaW1wbGlmaWVkLW9wZXJhdG9yLmgiCiNpbmNsdWRlICJzcmMvY29tcGlsZXIvdHlwZS1jYWNoZS5oIgojaW5jbHVkZSAic3JjL2ljL2NhbGwtb3B0aW1pemF0aW9uLmgiCiNpbmNsdWRlICJzcmMvb2JqZWN0cy9jZWxsLWlubC5oIgojaW5jbHVkZSAic3JjL29iamVjdHMvZmllbGQtaW5kZXgtaW5sLmgiCiNpbmNsdWRlICJzcmMvb2JqZWN0cy9maWVsZC10eXBlLmgiCiNpbmNsdWRlICJzcmMvb2JqZWN0cy9vYmplY3RzLWlubC5oIgojaW5jbHVkZSAic3JjL29iamVjdHMvc3RydWN0LWlubC5oIgojaW5jbHVkZSAic3JjL29iamVjdHMvdGVtcGxhdGVzLmgiCgpuYW1lc3BhY2UgdjggewpuYW1lc3BhY2UgaW50ZXJuYWwgewpuYW1lc3BhY2UgY29tcGlsZXIgewoKbmFtZXNwYWNlIHsKCmJvb2wgQ2FuSW5saW5lUHJvcGVydHlBY2Nlc3MoTWFwUmVmIG1hcCwgQWNjZXNzTW9kZSBhY2Nlc3NfbW9kZSkgewogIC8vIFdlIGNhbiBpbmxpbmUgcHJvcGVydHkgYWNjZXNzIHRvIHByb3RvdHlwZXMgb2YgYWxsIHByaW1pdGl2ZXMsIGV4Y2VwdAogIC8vIHRoZSBzcGVjaWFsIE9kZGJhbGwgb25lcyB0aGF0IGhhdmUgbm8gd3JhcHBlciBjb3VudGVycGFydHMgKGkuZS4gTnVsbCwKICAvLyBVbmRlZmluZWQgYW5kIFRoZUhvbGUpLgogIC8vIFdlIGNhbiBvbmx5IGlubGluZSBhY2Nlc3NlcyB0byBkaWN0aW9uYXJ5IG1vZGUgaG9sZGVycyBpZiB0aGUgYWNjZXNzIGlzIGEKICAvLyBsb2FkIGFuZCB0aGUgaG9sZGVyIGlzIGEgcHJvdG90eXBlLiBUaGUgbGF0dGVyIGVuc3VyZXMgYSAxOjEKICAvLyByZWxhdGlvbnNoaXAgYmV0d2VlbiB0aGUgbWFwIGFuZCB0aGUgb2JqZWN0IChhbmQgdGhlcmVmb3JlIHRoZSBwcm9wZXJ0eQogIC8vIGRpY3Rpb25hcnkpLgogIHN0YXRpY19hc3NlcnQoT0REQkFMTF9UWVBFID09IExBU1RfUFJJTUlUSVZFX0hFQVBfT0JKRUNUX1RZUEUpOwogIGlmIChtYXAub2JqZWN0KCktPklzQm9vbGVhbk1hcCgpKSByZXR1cm4gdHJ1ZTsKICBpZiAobWFwLmluc3RhbmNlX3R5cGUoKSA8IExBU1RfUFJJTUlUSVZFX0hFQVBfT0JKRUNUX1RZUEUpIHJldHVybiB0cnVlOwogIGlmIChtYXAub2JqZWN0KCktPklzSlNPYmplY3RNYXAoKSkgewogICAgaWYgKG1hcC5pc19kaWN0aW9uYXJ5X21hcCgpKSB7CiAgICAgIGlmICghVjhfRElDVF9QUk9QRVJUWV9DT05TVF9UUkFDS0lOR19CT09MKSByZXR1cm4gZmFsc2U7CiAgICAgIHJldHVybiBhY2Nlc3NfbW9kZSA9PSBBY2Nlc3NNb2RlOjprTG9hZCAmJgogICAgICAgICAgICAgbWFwLm9iamVjdCgpLT5pc19wcm90b3R5cGVfbWFwKCk7CiAgICB9CiAgICByZXR1cm4gIW1hcC5vYmplY3QoKS0+aGFzX25hbWVkX2ludGVyY2VwdG9yKCkgJiYKICAgICAgICAgICAvLyBUT0RPKHZlcndhZXN0KTogQWxsb3dsaXN0IGNvbnRleHRzIHRvIHdoaWNoIHdlIGh

**Review by Gutodotado24 — APPROVED** [review]

**Comment by papis801** [comment]
9

**Comment by papis801** [comment]
9

**Comment by papis801** [comment]
99

**Comment by papis801** [comment]
@

**Comment by papis801** [comment]
99

**Comment by papis801** [comment]
999

**Pull request merged**